src/index.js

```diff
@@ -50,7 +51,11 @@ also could be gravity being down.`)
           ? `\n\n  [Force] Booting on port ${PORT}... \n`
           : `\n\n  [Force] Started on ${APP_URL}. \n`
 
-      app.listen(PORT, "0.0.0.0", () => console.log(message))
+      const server = app.listen(PORT, "0.0.0.0", () => console.log(message))
```

Comment:
AFAICT this is the canonical way to access the `server` object created by Express?

Reply:
Yup, it returns an [`http.Server`](https://nodejs.org/api/http.html#http_class_http_server) instance under the hood.